db/migrate/20170329230309_create_requested_topics.rb

```diff
@@ -0,0 +1,9 @@
+class CreateRequestedTopics < ActiveRecord::Migration[5.0]
+  def change
+    create_table :requested_topics do |t|
```

Comment:
Naming is Hard :tm:

What do you think about `TopicRequests`? Something about the past-tense `requested` bugs me. What do you think?